automation_tools/__init__.py

```diff
@@ -425,9 +425,24 @@ def setup_scap_client():
     run('yum -y install puppet-foreman_scap_client', warn_only=True)
 
 
-def install_discovery_image():
-    """Task to install foreman discovery image."""
+def setup_foreman_discovery():
+    """Task to setup foreman discovery."""
+
+    admin_password = os.environ.get('ADMIN_PASSWORD')
+    if admin_password is None:
+        admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
+    template_url = os.environ.get('PXE_DEFAULT_TEMPLATE_URL')
+    host = env['host']
+    if template_url is None:
+        print('You should specify the PXE global default template URL')
+        sys.exit(1)
     run('yum -y install foreman-discovery-image', warn_only=True)
+    run('wget -O /var/tmp/template.txt {0}'.format(template_url))
```

Comment:
Instead of hard-coding a reference to `/var/tmp/template.txt`, can we use `mktemp` instead?